python/doc/theory/reliability_sensitivity/importance_form.rst

```diff
@@ -23,6 +23,8 @@ input components with respect to the realization of the event. They are often se
 of the impact of modeling the
 input components as random variables rather than fixed values. FORM importance factors are defined as follows.
 
+Let :math:`\mat{R}\in{\cS\cO}_n(\Rset)` be the special orthogonal group
+of square orthogonal matrices of dimension :math:`n` with determinant 1.
```

Comment:
The sentence is redundant. You should name the space and precise its definition after: 

Let :math:`\mat{R}\in{\cS\cO}_n(\Rset)` be the special orthogonal group which is the set of square matrices :math:`\mat{M} in \cM_n(\Rset)`  such that :math:`\mat{M} \Tr{\mat{M}} = I_n` and with dertminant equal to 1.

Reply:
Fixed at 267b7b8f5ec2dd5cd636181c1969bad3d85f5c94. Please tell me if that suits your requirement.

Reply:
ok!